dashboard/app/controllers/registrations_controller.rb

```diff
@@ -23,18 +23,22 @@ def update
         @user.update_without_password(update_params(params))
       end
 
+    account_update_response(successfully_updated, @user)
```

Comment:
Why pass in @user, since it's a member. More importantly (and I realize this is not part of your PR), why lookup @user on line 12 by current_user.id? First of all, it could just assign `@user = current_user`. But why do we need @user when we can just refer to `current_user` directly? That should be accessible everywhere (controller and views). 

The only reason I can think of doing the lookup is to implicitly render a 404 (from ActiveRecord::RecordNotFound) if there's no logged in user, but that won't even work as stated because it would instead be a NoMethodError trying to reference `id` on a nil current_user.